src/book.py

```diff
@@ -180,39 +217,79 @@ def tokenize(self, on, rem_stopwords=True, stopword_lang='english',
                     str.maketrans('', '', string.punctuation + '”“’'))
         return token
 
-    def snippet(self, length, on, non_cont=False, inplace=False):
+    def snippet(self, length, on, groups=1, non_cont=False, randomized=False, rem_stopwords=False, ret_as_arr=False, with_replace=True, random_seed=0, inplace=False):
         """
+        Returns snippets of char/words/sent of various length depending on input.
+        # of snippets = groups (Default=1). # of char/word/sent of snippets = length.
+        non-cont: (Default = False): If True, returns random char/word/sent from starting point
+        randomized: (Default = False): If True, chooses a random point to start making snippets
+        rem_stopwords: Same as in Tokenize
+        ret_as_array: (Default = False): Returns one BookText object by appending all groups of snippets, else returns
+                                         an array of BookText objects
+        with_replace: (Default = True): For non-contiguous case, makes sure there is no repetition. If turned off, different
+                                        snippets in a group might have repeated char/word/sent.
+        random_seed: When positive, generates the same snippets everytime
+        
         """
-        # TODO add ability for random seed
+        if (random_seed != 0):
```

Comment:
See comment above, change to `if (random_seed is not None):`

Reply:
Ok.